Static example does not compile

[Static example](https://github.com/envy/esp-knx-ip/blob/master/examples/static-config/static-config.ino) does not compile in Arduino IDE (`1.8.7`) with esp8266 board library version `2.3.0`. Same error for different boards selected.

Issue looks similar to what was in #13 
```
C:\SANITIZED\Arduino\esp_knx_static_example\esp_knx_static_example.ino: In function 'void setup()':

esp_knx_static_example:34:76: error: invalid conversion from 'callback_id_t {aka unsigned char}' to 'callback_fptr_t {aka void (*)(const __message&, void*)}' [-fpermissive]

   callback_id_t temp_cb = knx.callback_register("Read Temperature", temp_cb);

                                                                            ^

In file included from C:\SANITIZED\Arduino\esp_knx_static_example\esp_knx_static_example.ino:8:0:

C:\SANITIZED\Arduino\libraries\esp-knx-ip-master/esp-knx-ip.h:367:19: error:   initializing argument 2 of 'callback_id_t ESPKNXIP::callback_register(String, callback_fptr_t, void*, enable_condition_t)' [-fpermissive]

     callback_id_t callback_register(String name, callback_fptr_t cb, void *arg = nullptr, enable_condition_t cond = nullptr);

                   ^

esp_knx_static_example:35:71: error: invalid conversion from 'callback_id_t {aka unsigned char}' to 'callback_fptr_t {aka void (*)(const __message&, void*)}' [-fpermissive]

   callback_id_t hum_cb = knx.callback_register("Read Humidity", hum_cb);

                                                                       ^

In file included from C:\SANITIZED\Arduino\esp_knx_static_example\esp_knx_static_example.ino:8:0:

C:\SANITIZED\Arduino\libraries\esp-knx-ip-master/esp-knx-ip.h:367:19: error:   initializing argument 2 of 'callback_id_t ESPKNXIP::callback_register(String, callback_fptr_t, void*, enable_condition_t)' [-fpermissive]

     callback_id_t callback_register(String name, callback_fptr_t cb, void *arg = nullptr, enable_condition_t cond = nullptr);

                   ^

esp_knx_static_example:36:73: error: invalid conversion from 'callback_id_t {aka unsigned char}' to 'callback_fptr_t {aka void (*)(const __message&, void*)}' [-fpermissive]

   callback_id_t pres_cb = knx.callback_register("Read Pressure", pres_cb);

                                                                         ^

In file included from C:\SANITIZED\Arduino\esp_knx_static_example\esp_knx_static_example.ino:8:0:

C:\SANITIZED\Arduino\libraries\esp-knx-ip-master/esp-knx-ip.h:367:19: error:   initializing argument 2 of 'callback_id_t ESPKNXIP::callback_register(String, callback_fptr_t, void*, enable_condition_t)' [-fpermissive]

     callback_id_t callback_register(String name, callback_fptr_t cb, void *arg = nullptr, enable_condition_t cond = nullptr);

                   ^

exit status 1
invalid conversion from 'callback_id_t {aka unsigned char}' to 'callback_fptr_t {aka void (*)(const __message&, void*)}' [-fpermissive]
```

--- a/examples/static-config/static-config.ino
+++ b/examples/static-config/static-config.ino
@@ -31,14 +31,14 @@ void setup() {
   pinMode(LED_PIN, OUTPUT);
   Serial.begin(115200);
 
-  callback_id_t temp_cb = knx.callback_register("Read Temperature", temp_cb);
-  callback_id_t hum_cb =knx.callback_register("Read Humidity", hum_cb);
-  callback_id_t pres_cb =knx.callback_register("Read Pressure", pres_cb);
+  callback_id_t temp_cb_id = knx.callback_register("Read Temperature", temp_cb);
+  callback_id_t hum_cb_id =knx.callback_register("Read Humidity", hum_cb);
+  callback_id_t pres_cb_id =knx.callback_register("Read Pressure", pres_cb);
 
   // Assign callbacks to group addresses (2/1/1, 2/1/2, 2/1/3)
-  knx.callback_assign(temp_cb, knx.GA_to_address(2, 1, 1));
-  knx.callback_assign(hum_cb, knx.GA_to_address(2, 1, 2));
-  knx.callback_assign(pres_cb, knx.GA_to_address(2, 1, 3));
+  knx.callback_assign(temp_cb_id, knx.GA_to_address(2, 1, 1));
+  knx.callback_assign(hum_cb_id, knx.GA_to_address(2, 1, 2));
+  knx.callback_assign(pres_cb_id, knx.GA_to_address(2, 1, 3));
 
   // Set physical address (1.1.1)
   knx.physical_address_set(knx.PA_to_address(1, 1, 1));

